packages/itwinui-css/src/time-picker/time-picker.scss

```diff
@@ -11,13 +11,17 @@
   height: $iui-baseline * 26;
   display: inline-flex;
   background-color: var(--iui-color-background-1);
+  border-top-right-radius: $iui-border-radius;
+  border-bottom-right-radius: $iui-border-radius;
 
   &:not(:first-child) {
     border-left: 1px solid var(--iui-color-background-border);
   }
 
   &:first-child {
     box-shadow: $iui-elevation-2;
+    border-top-left-radius: $iui-border-radius;
+    border-bottom-left-radius: $iui-border-radius;
```

Comment:
Can we put lines 23 and 24 below line 15? I guess there is a reason why there is a separate `first-child` selector, but I'm not sure what that is

Reply:
i only want to apply these properties when the time-picker is the first-child.

in other words, i don't want the top-left and bottom-left corners to be rounded when the time-picker is used with the date-picker. see https://github.com/iTwin/iTwinUI/pull/746#discussion_r948985185